ts/components/services/ServiceSectionListComponent.tsx

```diff
@@ -76,6 +76,8 @@ class ServiceSectionListComponent extends React.Component<Props> {
       <RefreshControl refreshing={isRefreshing} onRefresh={onRefresh} />
     );
 
+    sections.sort((a, b) => (a.title || "").localeCompare(b.title));
```

Comment:
- avoid object mutation
- sections sorting is done on each state/props update (you are in render function) causing low performance and frames drop (section could have several items) . Try instead to change the caller component **ServicesScreen**: create a selector to get all sections by replacing the existing one https://github.com/teamdigitale/io-app/blob/548fff7b0803c4b313276d0695afc12f798476ff/ts/screens/preferences/ServicesScreen.tsx#L128-L140 keep sections into the component's state and check when it must be sorted again (_getDerivedStateFromProps_ could be useful)

Reply:
@Undermaken can you check the latest [commit](https://github.com/teamdigitale/io-app/pull/1153/commits/9c70888f4ccac64de7e2133b8bad26d264be92ec), please? Should it be done in that way?

Reply:
@Alex193a sorry for the delay. I will be off until next Tuesday. I will review asap. Until that keep this PR sync with master. Thank yoi ;)